src/xhr.ts

```diff
@@ -182,6 +184,8 @@ export class XhrModule {
             return handlePolling(finalUrl, finalSettings, this.ajax.bind(this));
         }
 
+        this.logPossiblyDeprecatedRestApiCall(response.headers);
```

Comment:
hmm just thinking if it would be better to split condition and log itself from this method...
```
if (isRestApiDeprectated(response.headers)) {
    logDeprecatedRestApiCall(...);
}
```

especially I don't like this "Possibly" word